packages/go/ecosystem/ecosystem.go

```diff
@@ -0,0 +1,29 @@
+package ecosystem
+
+import (
+	"strings"
+
+	"github.com/ossf/osv-schema/constants"
+)
+
+type Parsed struct {
```

Comment:
I wonder if there's a better name for this struct. I normally would want this to just be called Ecosystem, but `ecosystem.Ecosystem` also seems a bit weird.

Reply:
I wondered about that too but then actually really came around to it - I think it actually works really nicely with how Go recommends you do naming and that there shouldn't be any place outside of this package where the type won't be prefixed with `ecosystem`, thus it'll always be `ecosystem.Parsed`